config/schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "doc": "The application environment.",
+    "format": ["production", "development", "test"],
+    "default": "development",
+    "env": "NODE_ENV"
+  },
+  "akismet_antispam_api_key": {
```

Comment:
since we're easing into using https://standardjs.com/rules.html, shall we stick with camelcase for keys?